src/components/Resources/Quiz/Categories/QuizPage.jsx

```diff
@@ -37,6 +40,7 @@ export default function QuizPage() {
     const [clickedAnswerIndex, setClickedAnswerIndex] = useState(null);
     const [buttonClicked, setButtonClicked] = useState(false);
     const [disableSkipButton, setDisableSkipButton] = useState(true);
+    const [ShowAdditionalInfo, setShowAdditionalInfo] = useState(false);
```

Comment:
```suggestion
    const [showAdditionalInfo, setShowAdditionalInfo] = useState(false);
```
also in all places where you used this variable

Reply:
i did not understand your question

Reply:
I don't have question, change to lower case in all places that you use is variable.

Reply:
aa ok sorry